streams/src/main/java/org/apache/kafka/streams/kstream/KGroupedStream.java

```diff
@@ -438,7 +439,8 @@
      * query the value of the key on a parallel running instance of your Kafka Streams application.
      *
      * <p>
-     * For failure and recovery the store will be backed by an internal changelog topic that will be created in Kafka.
+     * For failure and recovery the store (which always will be of type {@link TimestampedKeyValueStore} -- regardless of what
+     * is specified in the parameter {@materialized}) will be backed by an internal changelog topic that will be created in Kafka.
```

Comment:
`{@materialized}` is not valid markup as far as I know. Should we `{@code materialized}`? (same below)

Reply:
Yes, actually, I wanted to write this but somehow... I will correct it. Shoudl I then squash the commits?

Reply:
Done

Reply:
No need to squash commits. We squash them automatically when merging.